<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="LP009302" failures="1" tests="3" name="Test" time="36.696" errors="0" timestamp="2023-05-06T20:14:45 IST">
  <testcase classname="testcases.LoginPageTest" name="verifyLoginPage" time="0.793"/>
  <testcase classname="testcases.LoginPageTest" name="verifyTitle" time="0.03">
    <failure type="java.lang.AssertionError" message="expected [Welcome To XTRAPOWER Fleet Card1] but found [Welcome To XTRAPOWER Fleet Card]">
      <![CDATA[java.lang.AssertionError: expected [Welcome To XTRAPOWER Fleet Card1] but found [Welcome To XTRAPOWER Fleet Card]
at testcases.LoginPageTest.verifyTitle(LoginPageTest.java:65)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verifyTitle -->
  <testcase classname="testcases.CustomerMenuTest" name="verifyAddCustomer" time="14.348"/>
  <testcase classname="base.BaseClass" name="@AfterSuite afterSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite afterSuite -->
  <testcase classname="base.BaseClass" name="@BeforeSuite loadConfig" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite loadConfig -->
</testsuite> <!-- Test -->
